README.md

```diff
@@ -13,10 +13,9 @@ Salesforce component is designed for Salesforce API integration.
 
 
 ### API version
-The component uses Salesforce - API Version 25.0, except:
+The component uses Salesforce - API Version 45.0, except:
 - Action: Lookup Object - API Version 39.0
```

Comment:
why Lookup Object wasn`t changed to v45?

Reply:
Updated